package.json

```diff
@@ -77,7 +78,7 @@
     "countly-sdk-nodejs": "^20.11.0",
     "electron-serve": "^1.1.0",
     "electron-store": "^8.1.0",
-    "electron-updater": "^5.3.0",
+    "electron-updater": "^6.2.1",
```

Comment:
:information_source:  This includes fix for `powershell.exe -NoProfile -NonInteractive` errors:
https://github.com/electron-userland/electron-builder/issues/7127#issuecomment-1991876763